contrib/container-build/scripts/fdbuild.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -xeo pipefail
+
+GITDIR=/build/src
+source "$HOME/.cargo/env"
+
+git clone --recurse-submodules https://github.com/firedancer-io/firedancer.git $GITDIR
+cd "$GITDIR"
+
+# Allow for a tag, release, or branch
+[ -n "$1" ] && git checkout "$1"
+
+# Currently needed to build and install openssl build libs that support QUIC
+./deps.sh install
+
+# Do the thing
+MACHINE=linux_gcc_x86_64 make -j fdctl solana
```

Comment:
Can we make `MACHINE` a parameter, with `linux_gcc_x86_64` as the default?

Reply:
added... with an additional check whether a specific machine type exists